packages/orbit-components/src/Card/index.tsx

```diff
@@ -46,7 +46,7 @@ export default function Card({
     <div
       id={id}
       className={cx(
-        "orbit-card font-base bg-white-normal [&>*]:border-elevation-flat-border-color lm:[&>*:first-child]:rounded-t-normal lm:[&>*:last-child]:rounded-b-normal w-full [&>*:first-child]:border-t",
+        "orbit-card font-base bg-white-normal *:border-elevation-flat-border-color lm:[&>*:first-child]:rounded-t-normal lm:[&>*:last-child]:rounded-b-normal w-full [&>*:first-child]:border-t",
```

Comment:
Can't the other `[&>*]` selectors be replaced as well?

Reply:
yes, sorry, I wasn't sure about it, bcs changing pseudoclasses to `*:first-child:...` didn't work, but I have found that's it's working in this format: `*-[:first-child]...` 🤗

It should be changed in this commit (tested with `Card` component in SB). https://github.com/kiwicom/orbit/pull/4402/commits/798c9382e9843c81703db029834b69468ea639cf

@DSil

Reply:
@DSil 
Nope,
I tried these variants:
- `*:first-child:...` ⛔ 
<img width="1266" alt="image" src="https://github.com/kiwicom/orbit/assets/19727168/5c195d2a-c19f-499b-9e79-1faed7d78fb7">

- `[*:first-child]:...` ⛔ 
<img width="1259" alt="image" src="https://github.com/kiwicom/orbit/assets/19727168/70345154-6222-4e65-a11e-e7f03a581735">

- `*-[:first-child]:...` ✅ 
<img width="1279" alt="image" src="https://github.com/kiwicom/orbit/assets/19727168/f88ebecd-c6ec-4af5-a3d0-62f0626f9987">

- `*[:first-child]:...` ⛔ 

<img width="1266" alt="image" src="https://github.com/kiwicom/orbit/assets/19727168/5cb28ceb-46ab-40c5-bc65-eb31e596d178">